### ISSUE TITLE
Nightscout does not sanitize private APNS keys for Loop remote overrides

### ISSUE THREAD

**Issue by jcantara** [nightscout/cgm-remote-monitor#6313]
**Describe the bug**
Nightscout currently does not sanitize the APNS private key when configured as here: https://loopkit.github.io/loopdocs/nightscout/remote-overrides/

The APNS key is used for loop "Remote Override" functionality, where Nightscout can send an Apple push notification to the loop app running on the looper's iPhone to enable or disable a temporary override remotely.

This private key is currently being exposed to the public when someone has configured Nightscout to use this remote overrides loop setting.

This private key could be used to craft a remote override command to any looper's iPhone that is exposing this key, allowing any random person to change the insulin delivery settings at will. This is a **_very bad thing_**.

**To Reproduce**
1. Configure nightscout with loop remote overrides, per here: https://loopkit.github.io/loopdocs/nightscout/remote-overrides/
2. Load nightscout root page, look at `status.json` request contents (don't need to be admin authorized, anonymous access also shows it)
3. See the APNS private key

**Expected behavior**
Should not see the private key

**Your setup information**
- Believe every version since remote overrides was merged will have this issue

**Additional context**
https://github.com/nightscout/cgm-remote-monitor/pull/6311

**Comment by sulkaharo** [comment]
This should be fixed in release 14, have you replicated this with the latest release? I just double checked and the settings sanitisation works for me on 14 as expected.

**Comment by sulkaharo** [comment]
Right, the screen capture in the ticket shows another method of getting hold of the settings on the site, let's get that fix in too.

**Comment by sulkaharo** [comment]
Oops sorry didn't mean to close this! I now double checked the fix in release 14 and it does fix both the previous issue with the status.json API endpoint as well as passing the sensitive settings into the client sandbox. I checked the unit test in your PR separately, and it passes when run with release 14 without your fix. Did you develop the fix against release 13, or can you point a way to repro the issue with release 14?

**Comment by jcantara** [comment]
Ah, my bad on that, the change I made was pre-14, and didn't think to check that someone else had already solved it :) (thank you!). Apologies for the faux-pas.

**Comment by sulkaharo** [comment]
I have to say though, this was one of the most valuable contributions in a long while, even if the issue was already resolved! The fact that you went through to implement a fix and report the issues is hugely appreciated, so thank you for that.
